packages/dd-trace/src/plugins/plugin.js

```diff
@@ -42,6 +42,12 @@ module.exports = class Plugin {
     storage.enterWith({ ...store, span })
   }
 
+  skip () {
```

Comment:
Can you add a comment of what this does? While it's clear when you know what it's doing, it could be confusing to an outside contributor.

Reply:
@bengl asked for the comment so I'll leave it to him to review the text.

Reply:
It should say something like:

> Prevents creation of spans here and all async descendants.